README.md

```diff
@@ -74,16 +101,68 @@ const response = await ezcap.invoke({method, capability, payload});
 const capability = 'https://zcap.example/foo';
 const delegate = 'did:key:z6MkpTHR8VNsBxYAAWHut2Geadd9jSwuBV8xRoAnwWsdvktH';
 const actions = ['read'];
-const delegatedCapability = ezcap.delegate({capability, delegate, actions});
+const delegatedCapability = zcapClient.delegate({capability, delegate, actions});
 ```
 
-## Contribute
+### Reading with a Delegated Capability
+
+```js
+const url = '/my-account/items/123';
+const capability = await getCapabilityFromDatabase({url}); // defined by your code
+
+// reading a URL using a zcap will result in an HTTP Response
+const response = await zcapClient.read({url, capability});
+
+// retrieve the JSON data
+const items = await response.json();
+```
+
+### Writing with a Delegated Capability
 
-See [the contribute file](https://github.com/digitalbazaar/bedrock/blob/master/CONTRIBUTING.md)!
+```js
+const url = '/my-account/items';
+const item = {label: 'Widget'};
+const capability = await getCapabilityFromDatabase({url}); // defined by your code
+
+// writing a URL using a zcap will result in an HTTP Response
+const response = await zcapClient.write({url, capability, json: item});
+
+// process the response appropriately
+const writtenItem = await response.json();
+```
+
+### Invoking with a Root Capability
```

Comment:
I'm assuming 'invoking' here means - making a general REST request using a zcap that's something other than read or write. In which case, I think it's a somewhat confusing term. Maybe it can be `zcapClient.request()`?

Reply:
I'm fine w/ the change... @dlongley, thoughts?

Reply:
Yeah `request` is fine by me.

Reply:
Done in 9612f0cee6d2886782424d7fc1a6845036d5d926 and e680b0862f9d723d48f5de317a2a9d2c1a466ebe.